test/e2e/util/fixture.go

```diff
@@ -206,6 +206,8 @@ func createClusterOpts(ctx context.Context, client crclient.Client, hc *hyperv1.
 		}
 
 		opts.BaseDomain = defaultIngressOperator.Status.Domain
+	case hyperv1.PowerVSPlatform:
+		opts.InfraID = fmt.Sprintf("%s-infra", hc.Name)
```

Comment:
for aws it is:
```golang
opts.InfraID = hc.Name
```
why it is different for powervs?

Reply:
Just wanted to differentiate name and infraID, added `-infra` suffix.